src/packages/shared-types/opensearch/main/transforms/seatool.ts

```diff
@@ -7,14 +7,17 @@ import {
   SeatoolOfficer,
 } from "../../..";
 
-import { PlanType } from "../../../planType";
+import {
+  Authority,
+  SEATOOL_AUTHORITIES,
+} from "shared-types";
 
-type AuthorityType = "SPA" | "WAIVER" | "MEDICAID" | "CHIP";
+type Flavor = "SPA" | "WAIVER" | "MEDICAID" | "CHIP";
```

Comment:
Can we come up with a better name than `Flavor`? This doesn't tell me anything about the categories of SPA, WAIVER, etc.